projects/scion/e2e-testing/src/dialog.po.ts

```diff
@@ -46,8 +46,15 @@ export class DialogPO {
     return fromRect(await this._dialog.boundingBox());
   }
 
-  public async getGlassPaneBoundingBox(): Promise<DomRect> {
-    return fromRect(await this.locator.page().locator('.cdk-overlay-pane.wb-dialog-glass-pane', {has: this.locator}).boundingBox());
+  public async getGlassPaneBoundingBoxes(): Promise<Set<DomRect>> {
+    const dialogId = await this.locator.getAttribute('data-dialogid');
+    const glassPaneLocators = await this.locator.page().locator(`div.e2e-glasspane[data-owner="${dialogId}"]`).all();
+
+    const boundingBoxes = new Set<DomRect>();
```

Comment:
Why Set?

Reply:
We do not know the order of the glass panes.